package.json

```diff
@@ -72,11 +71,12 @@
     "lodash": "^4.17.15",
     "messageformat": "^2.3.0",
     "query-string": "^7.1.1",
+    "regenerator-runtime": "^0.13.3",
```

Comment:
Hey @quad can see you have added `regenerator-runtime` and removed `@types/yup`? Any specific reason to do so?

Reply:
- I removed `@types/yup` because `yup` now provides its own type definitions. [`@types/yup` is deprecated.](https://www.npmjs.com/package/@types/yup)
- I added `regenerator-runtime` because it's an implicit dependency that, if missing, [breaks the build](https://app.circleci.com/pipelines/github/bigcommerce/checkout-sdk-js/9746/workflows/c1bdbcfa-2cfd-4ea2-9202-37953792e2de/jobs/30724)